lib/java/com/google/android/material/button/MaterialButtonToggleGroup.java

```diff
@@ -769,6 +775,22 @@ void onButtonCheckedStateChanged(@NonNull MaterialButton button, boolean isCheck
       checkInternal(button.getId(), isChecked);
   }
 
+  /**
+   * setEnable this {@link MaterialButtonToggleGroup} and all it's {@link MaterialButton} children
```

Comment:
nit: Enables this {@link MaterialButtonToggleGroup} and all its {@link MaterialButton} children

Reply:
Done